### PR TITLE
feat(plugins): manage multiple formats instead of xml only

### PR DESCRIPTION
**Pull request by jleveugle**

### CODE REVIEW

File: src/plugins/translation-inject.js [hunk 1]
```diff
@@ -17,14 +17,17 @@ module.exports = (opts = {}) => {
...
-          const match = text.match(/@ngTranslationsInject(.*)/);
+          const match = text.match(/@ngTranslationsInject:?([a-zA-Z]+)?(.*)/);
           if (match) {
             const translations = _.chain(match)
-              .get(1)
+              .get(2)
               .split(/\s+/)
               .value();
+
+            const format = match[1];
```
Comment (frenautvh) [inline src/plugins/translation-inject.js]: _.get(match, "1") for consistency

File: src/plugins/translation-inject.js [hunk 2]
```diff
@@ -17,14 +17,17 @@ module.exports = (opts = {}) => {
       const magicString = new MagicString(code);
       this.parse(code, {
         onComment: (block, text, start, end) => {
-          const match = text.match(/@ngTranslationsInject(.*)/);
+          const match = text.match(/@ngTranslationsInject:?([a-zA-Z]+)?(.*)/);
           if (match) {
             const translations = _.chain(match)
-              .get(1)
+              .get(2)
```
Comment (frenautvh) [inline src/plugins/translation-inject.js]: will it works if format is not provided ?

Comment (jleveugle) [reply src/plugins/translation-inject.js]: Yes, format will be `undefined` and by default, format is xml :)

Cf: https://github.com/ovh-ux/component-rollup-config/pull/22/files#diff-cb5d684f7dcca02f073d4210651e473fR72

File: src/plugins/translation-inject.js (hunk 1 above)
Comment (jleveugle) [reply src/plugins/translation-inject.js]: Yes, i saw the idea, but i think your suggestion is false ^^

Should be `_.get(match, '[1]')`, right ?

Comment (frenautvh) [reply src/plugins/translation-inject.js]: yep :)

**Pull request merged**